host/4/bullseye/amd64/node/node14/node14-composite.template

```diff
@@ -20,7 +20,7 @@ RUN apt-get update && \
 #    libdbus-1-3 libexpat1 libfontconfig1 libgbm1 libgcc1 libgconf-2-4 libgdk-pixbuf2.0-0 libglib2.0-0 \
 #    libgtk-3-0 libnspr4 libpango-1.0-0 libpangocairo-1.0-0 libstdc++6 libx11-6 libx11-xcb1 libxcb1 \
 #    libxcomposite1 libxcursor1 libxdamage1 libxext6 libxfixes3 libxi6 libxrandr2 libxrender1 libxss1 \
-#    libxtst6 ca-certificates fonts-liberation libappindicator1 libnss3 lsb-release xdg-utils wget
+#    libxtst6 ca-certificates fonts-liberation libayatana-appindicator3-1 libnss3 lsb-release xdg-utils wget
```

Comment:
code is still commented. Also, lets add this library on need basis - not all languages / Cx scenarios need it. 

Lets go ahead and add to node and dotnet images only. Lets see if Cx ask for python and dotnet-isolated